docs/migrate.rst

```diff
@@ -61,7 +61,7 @@ Metrics records for bulk request don't contain the ``bulk-size`` property anymor
 --include-filters and --exclude-filters affect all operations
```

Comment:
I'm sorry I did not point it out initially but my comment about `--include-tasks` and `--exclude-tasks` also applies to the heading here.